src/esp/sensor/CameraSensor.cpp

```diff
@@ -66,18 +67,9 @@ void CameraSensor::recomputeBaseProjectionMatrix() {
   recomputeProjectionMatrix();
 }  // CameraSensor::recomputeNearPlaneSize
 
-CameraSensor& CameraSensor::setProjectionMatrix(
-    gfx::RenderCamera& targetCamera) {
-  targetCamera.setProjectionMatrix(width_, height_, projectionMatrix_);
-  return *this;
-}
-
-CameraSensor& CameraSensor::setTransformationMatrix(
-    gfx::RenderCamera& targetCamera) {
-  CORRADE_ASSERT(!scene::SceneGraph::isRootNode(targetCamera.node()),
-                 "CameraSensor::setTransformationMatrix: target camera cannot "
-                 "be on the root node of the scene graph",
-                 *this);
+auto CameraSensor::computeTransformationMatrix() {
+  // target camera cannot be on the root node of the scene graph"
+  ASSERT(!scene::SceneGraph::isRootNode(renderCamera_->node()));
```

Comment:
You can do it either in this PR or in another PR:
try to convert all the ASSERT to CORRADE_ASSERT in the CameraSensor.cpp